src/panels/lovelace/hui-root.ts

```diff
@@ -187,7 +193,15 @@ class HUIRoot extends LitElement {
                     .hass=${this.hass}
                     .narrow=${this.narrow}
                   ></ha-menu-button>
-                  <div main-title>${this.config.title || "Home Assistant"}</div>
+                  <div main-title>
+                    ${this.config.title || "Home Assistant"}
+                    ${this.config.append_view_title ?
```

Comment:
This can stay like it is. If the Config Var is false or undefined this logic will not append

Reply:
So this is 👍 dont change